### PR TITLE
Create release notes for CNS on-prem 5.3.1

### PR DESCRIPTION
**Pull request by dobriak**

### CODE REVIEW

File: cns/5.0/rn/release-information/release-notes-5-3-1.adoc [hunk 1]
```diff
@@ -17,19 +17,19 @@ July 15, 2022
 
 === New Features
 
-* Better exception reporting. Report a connection exception between a PU and external network (within target networks) caused by missing TCP Fast Open enablement in a Next Generation Firewall in the data path.
+* Better exception reporting: Within your target networks, a missing TCP Fast Open enablement in the data path of a Next Generation Firewall reports a connection exception between the processing unit (PU) and the external network.
```
Comment (dobriak) [inline cns/5.0/rn/release-information/release-notes-5-3-1.adoc]: Connection exception is reported when there is a NGFW with missing TFO option between a PU and an external network.

**Review by rcaballeromx — APPROVED** [review]

**Comment by Pubs-MV** [comment]
@dobriak @rcaballeromx , we also need to add this new file to the book.yml. Can you please do that so I can merge and publish?

**Comment by rcaballeromx** [comment]
@Pubs-MV My bad, I just updated the PR to add the notes to the book. PTAL

**Comment by dobriak** [comment]
> 

@Pubs-MV created a PR for book.yaml https://github.com/PaloAltoNetworks/prisma-cloud-docs/pull/432

**Review by Pubs-MV — APPROVED** [review]

**Comment by rcaballeromx** [comment]
@dobriak You can close that PR, I updated this one.

**Pull request merged**